bsp_sedi/soc/intel_ish/pm/aon/aon_task.c

```diff
@@ -750,7 +790,7 @@ static void handle_reset(enum ish_pm_state pm_state)
 	disable_csme_csrirq();
 
 	/* power off main SRAM */
-	sram_power(0);
+	//sram_power(0);
```

Comment:
No need to power off SRAM banks? If so, suggest to remove the lines with comments.

Reply:
done.